misc/userscripts/readability-js

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+// 
+// # Description
+// 
+// Summarize the current page in a new tab, by processing it with the standalone readability
+// library used for Firefox Reader View.
+// 
+// # Prerequisites
+// 
+//   - NODE_PATH might be required to point to your global node libraries (e.g. /usr/lib/node_modules)
+//   - Mozilla's readability library (npm install -g https://github.com/mozilla/readability.git)
+//     NOTE: You might have to *login* as root for a system-wide installation to work (e.g. sudo -s)
+//   - jsdom (npm install -g jsdom)
+//   - qutejs (npm install -g qutejs)
+// 
+// # Usage
+// 
+// :spawn --userscript readability-js
+// 
+// One may wish to define an easy to type command alias in Qutebrowser's configuration file:
+// c.aliases = {"readability" : "spawn --userscript readability-js", ...}
+
+const Readability = require('readability');
+const qute = require('qutejs');
+const JSDOM = require('jsdom').JSDOM;
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+const HEADER = `
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1, text/html, charset=UTF-8" http-equiv="Content-Type">
+    </meta>
+    <title>%s</title>
+    <style type="text/css">
+        body {
+            margin: 30px auto;
+            max-width: 650px;
+            line-height: 1.4;
+            padding: 0 10px;
+        }
+        h1, h2, h3 {
+            line-height: 1.2;
+        }
+    </style>
+</head>`;
+const scriptsDir = path.join(process.env.QUTE_DATA_DIR, 'userscripts.html');
```

Comment:
Probably no `.html` here?

Reply:
whoops

Reply:
I keep re-requesting reviews by accident by clicking random buttons, sorry for  bugging you about such a small script when you got a backlog of other PRs!